rewrite/rewrite/python/format/normalize_line_breaks_visitor.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import annotations
+
+from typing import Optional, TypeVar, Union
+
+from rewrite import Tree, P, Cursor, list_map
+from rewrite.java import J, Space, Comment, TextComment
+from rewrite.python import PythonVisitor, PySpace, GeneralFormatStyle, PyComment
+from rewrite.visitor import T
+
+J2 = TypeVar('J2', bound=J)
+
+
+class NormalizeLineBreaksVisitor(PythonVisitor):
+    def __init__(self, style: GeneralFormatStyle, stop_after: Tree = None):
+        self._stop_after = stop_after
+        self._stop = False
+        self._style = style
+
+    def visit_space(self, space: Optional[Space], loc: Optional[Union[PySpace.Location, Space.Location]],
```

Comment:
Also here we will need an override for `visit_marker`, because the base visitor class doesn't know about specific markers and thus won't visit the `Space` of `TrailingComma`.

Reply:
Can we add a test case with a trailing comma for this visitor? I don't see the visit_marker override anywhere.